independent-publisher-connectors/Shopify Admin Store/README.md

```diff
@@ -0,0 +1,45 @@
+# Shopify Admin Store
+Manage a shopify store's customers, products, collections and customers using the Shopify Admin API . [Shopify Online Store API](https://shopify.dev/docs/admin-api/rest/reference/online-store)
+
+
+## Pre-requisites
+To use this connector, you need the following
+1. A Microsoft Power Apps or Power Automate plan with custom connector feature
+2. A Shopify store URL (e.g. storename.myshopify.com )
+3. Shopify client credentials - client id and client secret 
+
+###### How to get Client Credenitals
+* [Sign Up](https://partners.shopify.com/signup) for Shopify Partner Account.
+* [Log In](https://partners.shopify.com/organizations) with your account details and generate credentials by creating an app using: [Generate credentials from your partner dashboard](https://shopify.dev/tutorials/authenticate-a-public-app-with-oauth#generate-credentials-from-your-partner-dashboard) 
```

Comment:
@thatapiguy Can you please provide all the instructions in step-by-step with screenshots that can be understandable by everyone?